presto-main/src/main/java/com/facebook/presto/util/GraphvizPrinter.java

```diff
@@ -424,7 +424,7 @@ public Void visitValues(ValuesNode node, Void context)
         public Void visitEnforceSingleRow(EnforceSingleRowNode node, Void context)
         {
             printNode(node, "Scalar", NODE_COLORS.get(NodeType.PROJECT));
-            return null;
+            return node.getSource().accept(this, context);
```

Comment:
this PR already depends on 3 other PRs, I do not want to add another dependency for such minor change